packages/cli/src/tools/releaseChecker/releaseCacheManager.ts

```diff
@@ -42,17 +40,18 @@ function saveCache(name: string, cache: Cache) {
 function getCacheRootPath() {
   const cachePath = path.resolve(os.homedir(), '.react-native-cli', 'cache');
   if (!fs.existsSync(cachePath)) {
-    mkdirp(cachePath);
+    mkdirp(cachePath, () => {});
```

Comment:
Just wondering @thymikee. Shouldn't we use `mkdirp.sync` here? Also, it seemes `@types/mkdirp` aren't accurate because there is no need for noop function (https://github.com/substack/node-mkdirp/blob/master/index.js).

Reply:
Yup, we should use `mkdirp.sync` here